src/ray/core_worker/core_worker.cc

```diff
@@ -207,7 +207,7 @@ CoreWorker::CoreWorker(const WorkerType worker_type, const Language language,
             return std::shared_ptr<RayletClient>(
                 new RayletClient(std::move(grpc_client)));
           },
-          memory_store_, task_manager_,
+          memory_store_, task_manager_, raylet_id,
```

Comment:
Can we rename these to "local_raylet_id_" and "local_raylet_client_" now that we talk to multiple raylets?

Reply:
I don't know if it's really necessary, the CoreWorker doesn't talk to other nodes or raylets directly (it's encapsulated in the transport classes).

Reply:
Is there any downside? Seems like it makes things strictly more clear.